syntax = "proto3";

import "hq/dto/Security.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.request";
option java_outer_classname = "BrokerRequestProto";
message BrokerRequest {
	Security security = 1;
}
